code/modules/mining/machine_processing.dm

```diff
@@ -198,84 +214,100 @@
 	var/selected_uranium = 0
 	var/selected_iron = 0
 	var/selected_clown = 0
+	var/selected_titanium = 0
 	var/on = 0 //0 = off, 1 =... oh you know!
 
 /obj/machinery/mineral/processing_unit/process()
 	var/i

```

Comment:
you touch the shit code you clean the shit code, I'll chuck a token in for you if you do it though, cos this is a lot of shitcode.